Dockerfile.dev

```diff
@@ -1,9 +1,12 @@
 FROM node:19-buster-slim
 WORKDIR /iris-messenger/
 ENV NODE_OPTIONS=--openssl-legacy-provider
-RUN apt-get update && apt-get install -y python3 build-essential
 
-COPY package.json yarn.lock ./
+COPY package.json .
+COPY yarn.lock .
+
 RUN yarn
 
-CMD [ "yarn", "dev-docker" ]
\ No newline at end of file
+COPY . .
```

Comment:
Without copying the src, the container will fail unless the source in mounted in a volume in the docker-compose file or on the command line.